### DISCUSSION TITLE
I've signed up to your newsletter, how do i get your templates? I haven't gotten your templates?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#104] [Q&A] [answered]
How do i get the templates??? Read below!
If i'm already signed up and got the templates but want the latest versions??? See #108

**Comment by tallguyjenks** [comment, marked as answer]
When you sign up for the newsletter, when you **CONFIRM** your subscription via the email they should download automatically as a `.zip` file.

You signed up:

![image](https://user-images.githubusercontent.com/29872822/129821859-4a02b0a2-443b-4f2f-b8fa-02c550a9d0e3.png)

Check your email:

![image](https://user-images.githubusercontent.com/29872822/129821888-f9e08766-eacd-454c-ac04-d00989a8cbdd.png)

Click the button

Then you should get taken to a page on my website that looks like this:

![image](https://user-images.githubusercontent.com/29872822/137287799-30427850-7bfc-4fc8-9510-4615c9df1d17.png)

Click the link to be taken to the vault:

![image](https://user-images.githubusercontent.com/29872822/137287919-8d40686d-e15e-44d8-8a8e-df957aa97eff.png)

Then download it:

![image](https://user-images.githubusercontent.com/29872822/137287957-f4676e9a-0630-47ee-be83-efeeed933e07.png)

And you should get a download:

![image](https://user-images.githubusercontent.com/29872822/129821921-b7336f0d-e7b9-4dfb-b860-de2b15d37b85.png)

1. If you signed up through my website that subscription form doesn't advertise the templates, if you wanted them but signed up there send me an email indicating you used that method of sign up and want the templates
2. Check your Spam/Junk/Deleted folders
3. Check your downloads/default download folder for the `.zip` maybe you didn't catch it getting downloaded

**Reply by tallguyjenks** [reply]
# HOW TO USE THE THING YOU JUST DOWNLOADED

1. right click on the file and extract the contents (windows) or just double click on it (macOS). you will now have a new non-zipped folder
2. rename the folder what you want
3. place that folder where you want
4. go to obsidian and open a new vault
5. open a folder as a vault

![image](https://user-images.githubusercontent.com/29872822/130171682-9491609b-6c44-4272-afdb-9682866fc15c.png)

6. open your new folder in obsidian, activate the community plugins, CSS should be automatically applied
7. enjoy :)

**Comment by hsayed21** [comment]
### Could you share the last update from your workspace or your Custom CSS Theme ??

**Reply by tallguyjenks** [reply]
what update?

**Reply by hsayed21** [reply]
your current custom CSS theme

**Reply by tallguyjenks** [reply]
see #108 :)

**Comment by bogenschlag** [comment]
Huh. I’ve tried to subscribe, but Safari says “Too many redirects” when I click on the confirmation link. 🤷🏼‍♂️

<img width="604" alt="image" src="https://user-images.githubusercontent.com/5374341/204222665-6df1b34f-e39b-4c19-8a54-3f24bbda88e9.png">

**Reply by tallguyjenks** [reply]
try another browser, i dont use safari 🤷🏻

**Reply by bogenschlag** [reply]
Well … It’s not the browser. Right now I can’t even open your Website, at all. (Safari says “Load cannot follow more than 20 redirections”.)

Brave: 
![image](https://user-images.githubusercontent.com/5374341/204485163-83738f99-7dcd-433d-941c-ea463a6cba20.png)

Firefox: 
![image](https://user-images.githubusercontent.com/5374341/204485209-8e0e555c-c06a-476a-ad08-a854e4f88512.png)

Edge: 
![image](https://user-images.githubusercontent.com/5374341/204485528-738e1585-6951-44d3-b0fc-1fd90e536a31.png)

**Reply by tallguyjenks** [reply]
oh shoot, i see what you mean, i cant access my site either... well that's a problem

**Reply by tallguyjenks** [reply]
okay i think i fixed the problem, should work just fine now in any browser. i think it was a web hosting issue

**Comment by luz-giuliano** [comment]
Hi, I subscribed to your newsletter, but I didn't receive the confirmation e-mail. (I checked spam box and this is not there too). I'm using Gmail.
Thanks a lot.

**Reply by luz-giuliano** [reply]
Hi, I try with Microsoft Edge browser and now I receive the confirmation email. There is some problem with chrome browser that are note send the confirmation email.
